tests/testthat/test-5transformCounts.R

```diff
@@ -117,9 +112,16 @@ test_that("transformCounts", {
 
         expect_equal(
             as.matrix(assays(mia::transformCounts(tse, method = "clr", pseudocount = pseudonumber))$clr),
-            apply(as.matrix(relative), 2, FUN=function(x){
+            apply(as.matrix(relative) + pseudonumber, 2, FUN=function(x){
                 log(x) - mean(log(x))
             }))
+        
+        
+        tse2 <- relAbundanceCounts(tse)
+        expect_true(all(round(as.matrix(assays(mia::transformCounts(tse2, method = "clr", 
+                                                                    abund_values = "relabundance", pseudocount = pseudonumber))$clr) -
+                                  as.matrix(assays(mia::transformCounts(tse, method = "clr", pseudocount = pseudonumber))$clr),
+                              8) == 0))
 
         #############################################################
```

Comment:
How about also adding 

```
data("GlobalPatterns")
tse <- GlobalPatterns
tse1 <- relAbundanceCounts(tse)
tse1 <- transformCounts(tse1, abund_values = "relabundance", method = "clr")
tse2 <- transformCounts(tse, abund_values = "counts", method = "clr")
assays(tse1)$clr == assays(tse2)$clr
```

These should have the same default behavior. How to do it with pseudocount, perhaps need to be discussed.

Reply:
This will result in an error due to potential rounding errors. Calculating the difference, rounding it and checking whether it is zero is more stable

Reply:
Oh yes, that's fine. I primarily meant to check that this default behavior is the same. Test can be implemented in any way that works best.

Reply:
Already done. See line 120-124